### PR TITLE
Update licenses and license checker

### PR DESCRIPTION
**Pull request by awjh-ibm**
Added licenses missing that were found in https://github.com/hyperledger/composer/issues/1220

Updated license checker to allow for sh files to contain shebang

Added license checker to check all necessary files

### CODE REVIEW

**Comment by awjh-ibm** [comment]
Please do not merge until mark has approved the new license checker version

**Comment by cazfletch** [comment]
@awjh-ibm  the build says 2 files don't have the license, can you fix please

**Review by sstone1 — APPROVED** [review]
LGTM 👍

**Pull request merged**
